docs/docs/domain.mdx

```diff
@@ -6,9 +6,9 @@ abstract: The domain defines the universe in which your assistant operates. It s
 ---
 
 Here is a full example of a domain, taken from the
-[concertbot](https://github.com/RasaHQ/rasa/tree/main/examples/concertbot) example:
+[concertbot](https://github.com/RasaHQ/rasa/tree/main/examples/nlu_based/concertbot) example:
```

Comment:
can you add a regex here in [this section of `package.json`](https://github.com/RasaHQ/rasa/blob/59fa4fdaaff702cc5283872d592ed90bc558debc/docs/package.json#L113-L118) to ignore the URL + a FIXME pointing to a Jira task to remove the pattern from the list? That way we avoid the catch-22 and it won't be disrupting for engineers looking at failed CI builds ;)

Reply:
oh nice- thank you!